_layouts/resource.html

```diff
@@ -1,12 +1,81 @@
 ---
 layout: default
 ---
+
 <div class="container-fluid bg-white text-black blog">
-    <div class="container">
-        <div class="row justify-content-center py-4">
-            <div class="col-md-8">
-                {{content}}
+  <div class="container">
+    <div class="resource-content d-none" id="resource-content">
```

Comment:
Is this used at all? I don't see `d-none` being override for this section anywhere

Reply:
Yes, it's used by the js by removing the class to show the content.